01-basics.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Version Control with Mercurial
+subtitle: Automated Version Control
+minutes: 5
+---
+> ## Learning Objectives {.objectives}
+>
+> *   Understand the benefits of an automated version control system.
+> *   Understand the basics of how Mercurial works.
+
+We'll start by exploring how version control can be used
+to keep track of what one person did and when.
+Even if you aren't collaborating with other people,
+automated version control is much better than this situation:
+
+[![Piled Higher and Deeper by Jorge Cham, http://www.phdcomics.com](fig/phd101212s.gif)](http://www.phdcomics.com)
+
+"Piled Higher and Deeper" by Jorge Cham, http://www.phdcomics.com
+
+We've all been in this situation before: it seems ridiculous to have multiple nearly-identical versions of the same document. Some word processors let us deal with this a little better, such as Microsoft Word's "Track Changes" or Google Docs' version history.
```

Comment:
Minor niggle: would be good to be consistent about line length: either wrap or don't